coral-component-actionbar/src/scripts/ActionBarContainerCollection.js

```diff
@@ -46,6 +46,7 @@ class ActionBarContainerCollection extends SelectableCollection {
       child = this._host.children[i];
       if (
         !child.hasAttribute('disabled') &&
+        (child.firstElementChild && !child.firstElementChild.hasAttribute('disabled')) &&
```

Comment:
I think this would be the correct place to check that the first selection item inside child is not disabled https://github.com/adobe/coral-spectrum/blob/master/coral-component-actionbar/src/scripts/getFirstSelectableWrappedItem.js#L32

Reply:
moved condition where you suggested